flow/util/misc.py

```diff
@@ -199,3 +200,19 @@ def keys_used(self):
 def fullmatch(regex, string, flags=0):
     """Emulate python-3.4 re.fullmatch()."""
     return re.match("(?:" + regex + r")\Z", string, flags=flags)
+
+
+def roundrobin(*iterables):
+    # From: https://docs.python.org/3/library/itertools.html#itertools-recipes
+    "roundrobin('ABC', 'D', 'EF') --> A D E B F C"
+    # Recipe credited to George Sakkis
+    num_active = len(iterables)
+    nexts = cycle(iter(it).__next__ for it in iterables)
```

Comment:
See? It's cyclic. (See previous comment) 😉

Reply:
:D I'll change that.